src/chatlist.rs

```diff
@@ -366,7 +366,9 @@ impl Chatlist {
         let mut lastcontact = None;
 
         let lastmsg = if let Ok(lastmsg) = Message::load_from_db(context, lastmsg_id).await {
-            if lastmsg.from_id != DC_CONTACT_ID_SELF && chat.typ == Chattype::Group {
+            if lastmsg.from_id != DC_CONTACT_ID_SELF
```

Comment:
This is better replaced with an exhaustive match so we don't miss it when more types are added.

Reply:
yip, i will do a commit and change that line and the other, similar ones mentioned below.